### PR TITLE
Added TotalBytes to show-beacons and html-report

### PR DESCRIPTION
**Pull request by Hannnah1**
I made it so the total bytes per connection shows up in the show-beacons command and html-report.  builds and passes tests.

### CODE REVIEW

File: pkg/beacon/repository.go [hunk 1]
```diff
@@ -50,6 +50,7 @@ type Result struct {
 	data.UniqueIPPair `bson:",inline"`
 	Connections       int64   `bson:"connection_count"`
 	AvgBytes          float64 `bson:"avg_bytes"`
+	TB								int64    `bson:"total_bytes"`
```
Comment (fullmetalcache) [inline pkg/beacon/repository.go:53]: Looks like indention is a bit off here. Guessing a tabs/spaces issue?

Comment (fullmetalcache) [reply pkg/beacon/repository.go:53]: Would you be opposed to expanding TB out into TotalBytes? Just to cut down on the number of acronyms. Not a big deal, just a suggestion.

**Review by fullmetalcache — COMMENTED** [review]
Just some very minor comments. Looks good!

File: pkg/beacon/repository.go (hunk 1 above)
Comment (Hannnah1) [reply pkg/beacon/repository.go:53]: TotalBytes is taken. I'm not too familiar with go, but I assume I cannot reuse the variable name even if it's within a struct. I can definitely expand it though.

Comment (fullmetalcache) [reply pkg/beacon/repository.go]: Sorry, can you please clarify a bit? As long as there isn't another struct field with the same name in that struct then you should be able to use that name.

Comment (Hannnah1) [reply pkg/beacon/repository.go]: No you're right I just got confused. Everything mentioned above has been changed. :)

**Review by fullmetalcache — APPROVED** [review]
Looks great!!

**Pull request merged**